crates/rustc_codegen_spirv/src/attr.rs

```diff
@@ -276,9 +276,8 @@ impl CheckSpirvAttrVisitor<'_> {
                 },
 
                 SpirvAttribute::Entry(_) => match target {
-                    Target::Fn
-                    | Target::Method(MethodKind::Trait { body: true })
-                    | Target::Method(MethodKind::Inherent) => {
+                    Target::Fn |
+Target::Method(MethodKind::Trait { body: true } | MethodKind::Inherent) => {
```

Comment:
Shouldn't this be indented? Maybe Rustfmt isn't updated to handle or patterns.
```suggestion
                    Target::Fn |
                    Target::Method(MethodKind::Trait { body: true } | MethodKind::Inherent) => {
```